index.html

```diff
@@ -0,0 +1,252 @@
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </section>
+      <section class="card-works-1">
+        <img src="./img/Snapshoot-Portfolio-1.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Multi-Post Stories</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works">
+        <img src="./img/Snapshoot-Portfolio-2.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Tonic</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works-1">
+        <img src="./img/Snapshoot-Portfolio-3.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Multi-Post Stories</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+    </div>
+    <footer class="base-about-myself">
+      <section class="top">
+        <h1>
```

Comment:
Please also change this tag 🧐 See my other comment about `h1` tags for further explanation 😬

Reply:
I did

Reply:
Addressed ✔